specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1103 @@
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Start a run": {
+            "$ref": "./examples/startRun.json"
+          }
+        }
+      }
+    },
+    "/execution/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/experiments/{experimentName}/snapshotrun": {
+      "post": {
+        "tags": [
+          "Execution"
+        ],
+        "summary": "Start a run from a snapshot on a remote compute target.",
+        "description": "Starts an experiment run on the remote compute target using the provided definition.json file to define the run.\r\n            The code for the run is retrieved using the snapshotId in definition.json.",
+        "operationId": "Execution_StartSnapshotRun",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "name": "experimentName",
+            "in": "path",
+            "description": "The experiment name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "definition",
+            "in": "body",
+            "description": "A JSON run definition structure.",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/RunDefinition"
+            }
+          },
+          {
+            "name": "runId",
+            "in": "query",
+            "description": "A run id. If not supplied a run id will be created automatically.",
+            "required": false,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9][\\w-]{0,255}$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A snapshot run was successfully started.",
+            "schema": {
+              "$ref": "#/definitions/RunStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Start a snapshot run": {
+            "$ref": "./examples/startSnapshotRun.json"
+          }
+        }
+      }
+    },
+    "/execution/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/experiments/{experimentName}/cancel": {
+      "post": {
+        "tags": [
+          "Execution"
+        ],
+        "summary": "Cancel a run.",
```

Comment:
We shouldn't document the Execution_CancelRun API -- it's a deprecated in favor of Execution_CancelRunWithUri.

Reply:
done